src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.html

```diff
@@ -67,7 +67,20 @@
       <ng-container matColumnDef="label">
         <td *matCellDef="let group">
           @if (!group.groupedByEntity) {
-            <span>{{ group.label }}</span>
+            @if (group.label === undefined && group.isInvalidOption) {
+              <span class="not-defined-label" style="color: #d9534f"
+                >[invalid option]</span
+              >
+            } @else if (group.label === undefined) {
+              <span class="not-defined-label" i18n>not defined</span>
+            } @else {
+              <span
+                class="colored"
+                [style.background-color]="group.color || 'transparent'"
+              >
+                {{ group.label }}
+              </span>
+            }
           } @else {
             <app-entity-block [entityId]="group.id"></app-entity-block>
           }
```

Comment:
maybe we can use DisplayConfigurableEnumComponent here to handle the coloring? Less duplicated code and ensured UI consistency.